src/orgwarden/__main__.py

```diff
@@ -1,17 +1,63 @@
+import sys
+from typing import Annotated
 import typer
-from .repo_crawler import fetch_org_repos
+from orgwarden.audit import audit_repository
+from orgwarden.repository import Repository
+from orgwarden.repo_crawler import fetch_org_repos
 
 app = typer.Typer()
 
 
 @app.command()
-def main(org_name: str) -> None:
+def ls_repos(org_name: str) -> None:
+    """
+    Lists all public, non-forked repositories for the specified <org_name>
+    """
     repos = fetch_org_repos(org_name)
 
     print(f"~~~~~ Public repositories found for {org_name} ~~~~~")
     for repo in repos:
         print(f"{repo.org}/{repo.name} - {repo.url}")
 
 
+@app.command()
+def audit_repo(
+    repo_owner: str,
+    repo_name: str,
+    test_run: Annotated[bool, typer.Option(help="Used for mock testing.")] = False,
```

Comment:
This flag shouldn't be included. Let's discuss in a call what it would take to remove the need for this.